### PR TITLE
Update relatedObjects

### PR DESCRIPTION
**Pull request by tremes**
Updated our `ClusterOperator` relatedObjects.

## Categories

- [ ] Bugfix
- [ ] Enhancement
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Sample archive

No new data

## Documentation

No update.

## Unit Tests

No new test

## Privacy

Yes. There are no sensitive data in the newly collected information.

## References

https://issues.redhat.com/browse/???
https://bugzilla.redhat.com/show_bug.cgi?id=???
https://access.redhat.com/solutions/???

### CODE REVIEW

**Comment by tremes** [comment]
@wking Can you please check?

File: pkg/controller/status/status.go [hunk 1]
```diff
@@ -200,7 +200,12 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 	if len(c.namespace) > 0 {
 		existing.Status.RelatedObjects = []configv1.ObjectReference{
 			{Resource: "namespaces", Name: c.namespace},
-			{Group: "apps", Resource: "deployments", Namespace: c.namespace, Name: "insights-operator"},
+			{Resource: "secrets", Namespace: "openshift-config", Name: "pull-secret"},
+			{Resource: "secrets", Namespace: "openshift-config", Name: "support"},
```
Comment (tremes) [inline pkg/controller/status/status.go:204]: This one is kinda optional, but we try to read it so I think it makes sense to include it.

File: pkg/controller/status/status.go [hunk 2]
```diff
@@ -200,7 +200,12 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
...
+			{Resource: "serviceaccounts", Namespace: c.namespace, Name: "gather"},
+			{Resource: "serviceaccounts", Namespace: c.namespace, Name: "operator"},
+			{Resource: "services", Namespace: c.namespace, Name: "metrics"},
+			{Resource: "configmaps", Namespace: c.namespace, Name: "service-ca-bundle"},
```
Comment (tremes) [inline pkg/controller/status/status.go:208]: This one is used here https://github.com/openshift/insights-operator/blob/master/pkg/cmd/start/start.go#L20

File: pkg/controller/status/status.go [hunk 3]
```diff
@@ -200,7 +200,12 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 	if len(c.namespace) > 0 {
 		existing.Status.RelatedObjects = []configv1.ObjectReference{
 			{Resource: "namespaces", Name: c.namespace},
-			{Group: "apps", Resource: "deployments", Namespace: c.namespace, Name: "insights-operator"},
```
Comment (tremes) [inline pkg/controller/status/status.go:203]: Well I think we can keep this one...

Comment (wking) [reply pkg/controller/status/status.go:203]: You can keep it if you like, but deployments that live in referenced namespaces [are covered too][1].

[1]: https://github.com/openshift/oc/blame/13f3e4b73df9b0853979e4130adf756784afe96f/pkg/cli/admin/inspect/namespace.go#L18-L30

**Comment by rluders** [comment]
A simple change, just reading it and seems pretty good.
/lgtm

**Pull request merged**